include/logging/log_core.h

```diff
@@ -210,6 +211,26 @@ extern "C" {
 	)								    \
 	))
 
+#ifdef CONFIG_LOG_MINIMAL
+void log_minimal_hexdump_print(const char *data, size_t size);
+
+#define Z_LOG(_level, fmt, ...) do { \
+		if (Z_LOG_CONST_LEVEL_CHECK(_level)) { \
+			printk(fmt "\n", ##__VA_ARGS__); \
```

Comment:
i think that i would be good to print log level as well.

Reply:
how does this sound: a set of configs where you could choose to have no text added, or selectively enable colors/timestamps/log module/func name/etc, and have it apply to both log frontends, with different defaults.